docs/Submit_a_new_ontology.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: doc
-id: Policy_for_OBO_namespace_and_associated_PURL_requests
-title: Policy for OBO IDSPACE and associated PURL requests
+id: Submit_a_new_ontology
+title: Submit a New Ontology to the OBO Foundry
 ---
 
 Please raise any questions or concerns on our GitHub tracker: https://github.com/OBOFoundry/OBOFoundry.github.io/issues
```

Comment:
Change to "For a simple overview of how to submit your ontology for consideration, please go [here](URL of new submit an ontology page)"

Reply:
I think we should move the Background section to the end.

Reply:
i made these changes.

the URL will need to be updated though after the new submit an ontology page PR is merged #1283